src/aws_sync.js

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env node
```

Comment:
this is typically only necessary in a script we're going to call from the command line - I think we only need it in our `index.js` file